guides/common/modules/ref_examples-of-organization-management-in-project.adoc

```diff
@@ -0,0 +1,24 @@
+[id="examples-of-organization-management-in-project"]
```

Comment:
technically "missing" `_{context}` as per our convention but I am totally OK.

```suggestion
[id="examples-of-organization-management-in-{project-context}"]
```

alternative: `examples-of-organization-management`

Reply:
it would probably be safer to include `_{context}`.

Reply:
Apart from missing {context}, the ID and title didn't match. Updated.